src/GeositeFramework/js/SubRegion.js

```diff
@@ -146,7 +146,15 @@
         _.each(subregions.areas, function(subregion) {
             var geom = new esri.geometry.Polygon(subregion.shape);
             var symbol = new esri.symbol.SimpleFillSymbol();
+            var outline = new esri.symbol.SimpleLineSymbol(
+                    esri.symbol.SimpleLineSymbol.STYLE_SOLID,
+                    new dojo.Color(subregion.outlineColor || subregions.outlineColor), 2);
```

Comment:
It defaults to a very faint white:

![selection_067](https://cloud.githubusercontent.com/assets/1042475/6802887/3abac262-d20a-11e4-933b-e5abf605beb2.png)